src/api/users.ts

```diff
@@ -0,0 +1,9 @@
+import { api } from ".";
```

Comment:
import path looks wrong

should be something like 

import { api } from '../api'

Reply:
done